src/main/java/me/mrCookieSlime/Slimefun/hooks/github/Contributor.java

```diff
@@ -16,7 +16,7 @@
 	
 	private String name;
 	private String profile;
-	private Optional<String> headTexture;
+	private Optional<String> headTexture = Optional.empty();
```

Comment:
The optional was intentionally left null.

Null = The texture has not been retrieved yet (or tooManyrequests, so try next time.
Empty = No mc account with that name exists.

Reply:
NotLikeThis